tests/pytest/test_regression.py

```diff
@@ -63,6 +63,7 @@ def get_n_processes(config):
 @pytest.mark.parametrize(
     ("config_filename", "tag"), 
     [
+        ("default.yml", "{version}-debug"),
```

Comment:
Don't you need to add the regression data for this as well?

Reply:
The results are bit-reproducible in debug mode vs. non-debug mode.

Reply:
The actually test is that the executable runs through successfully in debug mode, but it turns out that the results are bit-reproducible with debug mode on using the GNU compiler, so no need to use separate references.